File: code/a6130.pas
program Bai130;
var n, c, d :byte;
begin
  readln(n);
  d := n mod 10;
  while n > 0 do 
    begin
      c := n mod 10;
      if c <> d then  
      begin
      writeln('YES');
      exit;
      end;
      n := n div 10;
      end;
    writeln('NO');
end.

File: code/a6130.cpp
#include <iostream>
using namespace std;

int main() 
{
    int n,c,d;
    cin>>n;
    d=n/10;
    c=n%10;
    if ((c%2 + d%2)==1)
        cout<<"YES";
    else cout<<"NO";
    return 0;
}